### DISCUSSION TITLE
Mirroring a populated S3

### DISCUSSION THREAD

**Discussion by tofijak** [strapi/strapi#7349] [Legacy Topics]
Hey,

Is it possible to mirror a populated S3? 
Basically: If any file is uploaded to the S3 bucket from another source than Strapi it will appear in the Media Library the next time a user visit it.

Cheers,

Simon

**Comment by derrickmehaffy** [comment]
Currently that is not possible as during the file upload process, much of the metadata is added to the database at this time, along with a link to the file within the S3 bucket. Files added to S3 that were not done with Strapi means that Strapi is not aware of them (nothing in the database to point to the fact the file exists.)
